tutorials/katacoda/3-eventing-intro-channels/step2.md

```diff
@@ -0,0 +1,93 @@
+## Event Topologies
+We will walk through the simplest to more complicated topologies
+
+### 1:1 Event Delivery
+The most straightforward use case is that whenever events are produced, you want some code to handle that event.
+
+![1to1](./assets/1to1.png)
```

Comment:
Really like the diagram pictures ❤️ 

Wondering if we can make diagrams for serving tutorial

Reply:
Thanks! Will consider in a follow on PR